spec/defines/server_export_spec.rb

```diff
@@ -3,7 +3,14 @@
 describe 'nfs::server::export', :type => 'define' do
 
   context "nvs_v4 => false" do
-    let(:facts) {{ :operatingsystem => 'ubuntu', :concat_basedir => '/dne', :osfamily => 'debian', :clientcert => 'test.host' }}
+    let(:facts) {{
+      :operatingsystem => 'Ubuntu',
+      :osfamily => 'Debian',
+      :operatingsystemmajrelease => '12.04',
+      :concat_basedir => '/dne',
+      :osfamily => 'debian',
```

Comment:
This line is causing the failure - it is overwriting 'Debian' - that lowercase 'debian' is not hitting in params.pp.